### PR TITLE
removed depwarns

### PR DESCRIPTION
**Pull request by SimonDanisch**
I just did a search and replace and couldn't run the tests locally and travis is not set up as pointed out in randyzwitch/Twitter.jl/issues#3.
I hope someone else can run the tests before merging :)

Best,
Simon

### CODE REVIEW

**Comment by randyzwitch** [comment]
I'll review this more in-depth in the next day or two. Automated testing isn't setup yet because I don't want to get my main account blocked for excessive API calls in rapid succession :)

File: tests/tests.jl [hunk 1]
```diff
@@ -1,337 +1,6 @@
+using Twitter, DataFrames
 #Authentication using consumer & oauth values
-twitterauth("6nOtpXmf...",
```
Comment (randyzwitch) [inline tests/tests.jl]: If I'm reading this correctly, you deleted the tests, only leaving your credentials behind. You probably want to change your credentials after revoking this change.

Comment (randyzwitch) [inline tests/tests.jl]: While you're making changes, can you fix the REQUIRES file too? It has the 0.4- syntax.

**Comment by SimonDanisch** [comment]
Whoops, commited too hastily...Should be fixed!

**Comment by randyzwitch** [comment]
Did a few dozen tests, seems to be working and I don't get any warnings on load. Merging, then I'll make a change for pre-compilation, then tag a new release.

Thanks for making these changes, along with OAuth!

**Pull request merged**